tests/lean/inductionErrors.lean.expected.out

```diff
@@ -26,11 +26,11 @@ y : Nat
 inductionErrors.lean:50:2-50:16: error: alternative 'cons' is not needed
 inductionErrors.lean:53:2-55:16: error: alternative for 'Vec.cons' is not needed
 inductionErrors.lean:60:2-60:40: error: invalid alternative name 'upper2'
-inductionErrors.lean:66:23-66:28: warning: declaration uses 'sorry'
-inductionErrors.lean:65:29-65:34: warning: declaration uses 'sorry'
-inductionErrors.lean:72:29-72:34: warning: declaration uses 'sorry'
-inductionErrors.lean:71:29-71:34: warning: declaration uses 'sorry'
+inductionErrors.lean:66:23-66:23: warning: declaration uses 'sorry'
+inductionErrors.lean:65:29-65:29: warning: declaration uses 'sorry'
+inductionErrors.lean:72:29-72:29: warning: declaration uses 'sorry'
+inductionErrors.lean:71:29-71:29: warning: declaration uses 'sorry'
 inductionErrors.lean:74:2-74:34: error: unused alternative
-inductionErrors.lean:79:29-79:34: warning: declaration uses 'sorry'
-inductionErrors.lean:78:29-78:34: warning: declaration uses 'sorry'
+inductionErrors.lean:79:29-79:29: warning: declaration uses 'sorry'
+inductionErrors.lean:78:29-78:29: warning: declaration uses 'sorry'
 inductionErrors.lean:80:2-80:53: error: unused alternative
```

Comment:
It seems we lost information here. That is, the begin and end positions are equal. I am seeing the same behavior in many other tests.